packages/bpk-component-autosuggest/src/BpkAutosuggest.module.scss

```diff
@@ -153,4 +153,14 @@ $arrow-size: bpk-spacing-sm() * 3;
       @include bpk-caption;
     }
   }
+
+  &__section-title {
+    padding: bpk-spacing-lg() 0 $bpk-spacing-xs $bpk-spacing-md;
+    background-color: $bpk-color-white;
+    color: $bpk-text-secondary-day;
+
+    @include bpk-text;
+    @include bpk-text-sm;
```

Comment:
`bpk-text--bold` is deprecated, can you use instead the typography style equivalent to sm + bold?